ArduPlane/Log.cpp

```diff
@@ -52,7 +52,7 @@ void Plane::Log_Write_Attitude(void)
     logger.Write_POS();
 }
 
-// do logging at loop rate
+// do logging at 25 Hz
```

Comment:
I'm not a great fan of putting numbers on comments like this.  They get out of date.

I'm not even sure the code's not wrong; Copter logs this a lot faster, so why don't we do so for Q-modes?

Reply:
So for Copter the ATTITUDE_FAST logging is in a 400 Hz loop. For Plane it is marked in the scheduler as 25 Hz loop. It would make sense for Q-modes to have faster logging. Probably not necessary for non-Q-modes. I could try to rework Plane to be similar to Copter. Maybe add an ifdef for Qplane and non Qplane? This was dropped from 50 Hz to 25 Hz years ago. 25 Hz makes a lot of sense for normal planes.

Reply:
Agree you should have 400hz logging for Qplane. Might be easier to have make the fast loop 400hz but only enabled on qplane and then continue to log the other one at 25hz